Sources/TestSupport/Mocks/MockProjectProvider.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import Path
+import Testing
+import XcodeGraph
+import XcodeProj
+
+@testable import XcodeProjToGraph
+
+public struct MockWorkspaceProvider: WorkspaceProviding {
```

Comment:
That’s really cool and definitely a time saver! I wonder if it’s the right fit here, though. Since we only have a few simple mocks, adding a dependency with swift-syntax introduces more complexity and significant build-time overhead than needed. That said, I’m not against it. Just want to make sure we’re fully aware of the trade-offs.